theme/documentation/new_release_script.md

```diff
@@ -0,0 +1,11 @@
+# New Release Script
+
+The new release script, located in the root directory of the theme folder, will automate a few manual tasks for updating Help Center at the time of a new Liferay product release.
+
+## Automated Tasks
+
+- Running the script will automatically generate a new landing page template for the product. A content creactor can then go into the template to update its content.
```

Comment:
There is a typo. `creator`

Reply:
Thanks! Fixed and squashed with a previous commit.